test/thermo/RedlichKwongMFTP_Test.cpp

```diff
@@ -0,0 +1,165 @@
+#include "gtest/gtest.h"
+#include "cantera/thermo/RedlichKwongMFTP.h"
+#include "cantera/thermo/ThermoFactory.h"
+
+
+namespace Cantera
+{
+
+class RedlichKwongMFTP_Test : public testing::Test
+{
+public:
+    RedlichKwongMFTP_Test() {
+        test_phase.reset(newPhase("../data/co2_RK_example.cti"));
+    }
+
+    //vary the composition of a co2-h2 mixture:
+    void set_r(const double r) {
+        vector_fp moleFracs(7);
+        moleFracs[0] = r;
+        moleFracs[2] = 1-r;
+        test_phase->setMoleFractions(&moleFracs[0]);
+    }
+
+    std::unique_ptr<ThermoPhase> test_phase;
+};
+
+TEST_F(RedlichKwongMFTP_Test, construct_from_cti)
+{
+    RedlichKwongMFTP* redlich_kwong_phase = dynamic_cast<RedlichKwongMFTP*>(test_phase.get());
+    EXPECT_TRUE(redlich_kwong_phase != NULL);
+}
+
+TEST_F(RedlichKwongMFTP_Test, chem_potentials)
+{
+    test_phase->setState_TP(298.15, 101325.);
+
+    const double expected_result[9] = {
+        -4.573578067074649e+008,
+        -4.573471163377696e+008,
+        -4.573375748803425e+008,
+        -4.573290065058332e+008,
+        -4.573212695326964e+008,
+        -4.573142485189869e+008,
+        -4.573078484551440e+008,
+        -4.573019904340246e+008,
+        -4.572966083775078e+008
+    };
```

Comment:
Can you add something about why this is the expected result? I assume that this is a regression test based on running this calculation with the model, but it would be good to clarify that, and if possible explain why this is the expected trend in the chemical potential for CO2 as the composition changes.